UnityServer/functions/.gitignore

```diff
@@ -0,0 +1,10 @@
+## Compiled JavaScript files
```

Comment:
maybe we can combine what's in this .gitignore with the top level .gitignore?

Reply:
sure, will do